core/storage/changes_trie/impl/storage_changes_tracker_impl.cpp

```diff
@@ -21,11 +21,13 @@ namespace kagome::storage::changes_trie {
 
   StorageChangesTrackerImpl::StorageChangesTrackerImpl(
       std::shared_ptr<storage::trie::PolkadotTrieFactory> trie_factory,
-      std::shared_ptr<storage::trie::Codec> codec)
+      std::shared_ptr<storage::trie::Codec> codec,
```

Comment:
I guess it is not using

Reply:
It's not a code from this PR, but I checked and it is used.